frame/assets/src/lib.rs

```diff
@@ -1132,6 +1211,7 @@ pub mod pallet {
 
 			Asset::<T, I>::try_mutate(id, |maybe_asset| {
 				let mut asset = maybe_asset.take().ok_or(Error::<T, I>::Unknown)?;
+				ensure!(asset.status == AssetStatus::Live, Error::<T, I>::AssetNotLive);
```

Comment:
I would actually remove this check and additionally add `status` to the inputs. That way governance would be the one way to revert a `Destroying` asset to `Live`.

Reply:
Could you explain a bit more what you mean to "add status to the inputs"
"

Reply:
```rust
pub fn force_asset_status(
	origin: OriginFor<T>,
	#[pallet::compact] id: T::AssetId,
	owner: AccountIdLookupOf<T>,
	issuer: AccountIdLookupOf<T>,
	admin: AccountIdLookupOf<T>,
	freezer: AccountIdLookupOf<T>,
	#[pallet::compact] min_balance: T::Balance,
	is_sufficient: bool,
	is_frozen: bool,
	asset_status: AssetStatus,
) -> DispatchResult
```